### PR TITLE
Article page styles

### PR DESCRIPTION
**Pull request by vstanyshevskyy**
Restyled article page and navigation

### CODE REVIEW

File: src/templates/articles/index.js [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+  const post = Object.assign(
+    {
+      html: articleData.article.html,
+      next: articleData.next ? articleData.next.data : null,
+      previous: articleData.previous ? articleData.previous.data : null
+    },
+    ...articleData.article.data
+  );
+  const moreFromCategory = allArticles.filter(a =>
+    a.article.data.category === post.category
```
Comment (zoryana94) [inline src/templates/articles/index.js]: you can move to the const
```
const { category, title } = a.article.data;
return category === post.category...
```

File: src/templates/articles/index.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+                        <strong>Наступна стаття</strong>
+                        <span className="g1-gamma g1-gamma-1st">{post.next.title}</span>
+                      </Link>
+                    </li> :
+                    null
+                }
+              </ul>
+            </div>
+          </nav>
+          { moreFromCategory.length ?
```
Comment (zoryana94) [inline src/templates/articles/index.js]: length > 0
in React It will be true even if the array is empty

Comment (vstanyshevskyy) [reply src/templates/articles/index.js]: No, it's ok. https://vpershe.netlify.com/articles/video-art/

File: src/templates/articles/index.js (hunk 1 above)
Comment (vstanyshevskyy) [reply src/templates/articles/index.js]: thank you!

File: src/templates/articles/index.js (hunk 2 above)
Comment (zoryana94) [reply src/templates/articles/index.js]: sorry, I haven't noticed you use the map... yeah, it's ok... you just don't display the items

Comment (vstanyshevskyy) [reply src/templates/articles/index.js]: This check has nothing to do with map. In js and react [].length returns 0 and 0 is a falsy value.

**Pull request merged**